extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/ParquetExtensionsModule.java

```diff
@@ -34,8 +34,8 @@
   @Override
   public List<? extends Module> getJacksonModules()
   {
-    return Arrays.asList(
-        new SimpleModule("ParuqetInputRowParserModule")
+    return Collections.singletonList(
+        new SimpleModule("ParquetInputRowParserModule")
```

Comment:
Is it fully compatible? Couldn't somebody depend on the actual name of the module somehow?